docker-storage-setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #--
-# Copyright 2014-2015 Red Hat, Inc.
+# Copyright 2014-2016 Red Hat, Inc.
```

Comment:
Should we update existing line or add another one for 2016. I am not sure typically what's the convention.